3737/2/helpers/photo_helper.rb

```diff
@@ -0,0 +1,24 @@
+require 'net/http'
+require 'open-uri'
+
+# module that download photo
+module PhotoHelper
+  TOKEN = ENV['TOKEN']
+  API = "https://api.telegram.org/bot#{TOKEN}/getFile?file_id=".freeze
+  DOWNLOAD_API = "https://api.telegram.org/file/bot#{TOKEN}/".freeze
+
+  def photo_path
+    JSON.parse(URI.open(API + payload['photo'].last['file_id'])
```

Comment:
rather complex line
please split it in several
file_id =
json_response = URI.open("#{API}/#{file_id}").read
JSON.parse(json_response, symbolize_names: true).dig(:result, :file_path)